sentry.go

```diff
@@ -84,9 +85,19 @@ func (so *SentryOutput) Run(or pipeline.OutputRunner, h pipeline.PluginHelper) (
 		client *raven.Client
 	)
 
+	if or.Encoder() == nil {
+		return errors.New("Encoder required.")
+	}
+
 	sentryMsg := &SentryMsg{}
 
 	for pack = range or.InChan() {
+		if _, e = or.Encode(pack); e != nil {
```

Comment:
@mohit we use the return value from Encode now, and since PayloadDecoder is used by default, this preserves the old behavior of just using the payload, while also allowing us to use decoders that don't necessarily mutate the payload.
2edd897